pom.xml

```
@@ -62,8 +62,8 @@
     <dependencies>
       <dependency>
         <groupId>io.jenkins.tools.bom</groupId>
-        <artifactId>bom-2.361.x</artifactId>
-        <version>1836.vfe602c266c05</version>
+        <artifactId>bom-2.440.x</artifactId>
+        <version>2982.vdce2153031a_0</version>
```

Tests pass without requiring a newer Jenkins minimum version (at least on my Linux computer).  Is there something that prompts the change to a newer Jenkins minimum version when this is an API plugin and would generally keep a much older minimum Jenkins version?
```suggestion
        <artifactId>bom-2.361.x</artifactId>
        <version>1836.vfe602c266c05</version>
```